packages/site/src/pages/index.astro

```diff
@@ -40,7 +40,7 @@ const resourceCards = FRAMEWORK_RESOURCES().filter(
 <head>
 	{`<script type="application/ld+json">${JSON.stringify(searchableWebsiteStructuredData)}</script>`}
 </head>
-<BaseLayout title={`Featured ${formatFieldName(import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK)} resources`} className={scrollable}>
+<BaseLayout title="Featured resources" className={scrollable}>
```

Comment:
See comment above about the page title changes for the full scope of this change.